Documentation/getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting started
+
+## Before starting
+
+The Adobe Experience Platform Edge Network extension has the following peer dependencies, which must be installed to use the Edge Network extension:
+- [AEPCore](https://github.com/adobe/aepsdk-core-ios#readme)
+- [AEPEdgeIdentity](https://github.com/adobe/aepsdk-edgeidentity-ios#readme)
+
+## Configure the Edge Network extension in Data Collection UI
+1. Log into [Adobe Experience Cloud](https://experience.adobe.com) and select **Data Collection** under **Quick Access**.
+2. From **Tags**, locate or search for your Tag mobile property.
+3. In your mobile property, select **Extensions** tab.
+4. On the **Catalog** tab, locate or search for the **Adobe Experience Platform Edge Network** extension, and select **Install**.
+5. Select the **Datastream** you would like to use per environment. Read more about [datastreams](#datastreams) below.
+6. Set up the **Domain configuration** by either using the automatically populated domin, or a first party domain mapped to an Adobe-provisioned Edge Network domain. For more information, see [domain configuration](#domain-configuration) below.
+7. Select **Save**.
+8. Follow the [publishing process](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property#publish-the-configuration) to update SDK configuration.
+
+### Datastreams
+
+If no datastream was previously created, see [Configure datastreams](https://aep-sdks.gitbook.io/docs/getting-started/configure-datastreams) before moving to the next step.
+
+Select the desired datastream per each environment from the corresponding drop-down lists. If your organization uses multiple sandboxes, select the **Sandbox** first, then select the **Datastream** for each environment.
+
+The datastream used by the client-side implementation is one of the following:
+
+- the _Production_ environment configuration when the Data Collection Tags library is published to production (in the Published column in the publishing flow).
+- the _Staging_ environment configuration when the Data Collection Tags library is published to staging (in the Submitted column in the publishing flow).
+- the _Development_ environment configuration when the Data Collection Tags library is in development.
```

Comment:
the _Development_ environment configuration when the Data Collection Tags library is in development (in the Development column in the publishing flow).